src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -167,8 +180,10 @@ export default class RegistrationDefineBusiness extends Mixins(CommonMixin) {
   @Action setRegistrationDefineBusinessValid!: ActionBindingIF
   @Action setFolioNumber!: ActionBindingIF
   @Action setRegistrationBusinessNumber!: ActionBindingIF
+  @Action setRegistrationBusinessTypeConfirm!: ActionBindingIF
 
   // local variables
+  businessTypeConfirmValid = false
```

Comment:
This is good.

I just wanted to mention: do you remember when we looked at this initial value together and it didn't seem to be used? You definitely need to initialize local variables so that Vue makes them reactive. You cannot assign undefined here, but null would be OK. But false is even better as it sets the type of this local variable. An alternative would be `businessTypeConfirmValid: boolean = null` so that it has "no value" until something else sets it, but honestly the current code looks cleaner to me.

Reply:
I remember the method findIndex(f => !f), which is quite powerful to handle an array in which there is null and turn it into false.

Reply:
That too but I'm referring to the time when we were looking at code together and wondering why businessTypeConfirmValid was null (in the validation array) when it's clearly initialized to false here.

Anyway, I just wanted to mention proper init here is good, even though the valid is almost immediately set by the sub-component.